vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
But wasn't this added at our behest at not wanting to use a non-encrypted transport, meanwhile at the time the library only supported HTTP, not HTTPS.